contracts/core/Controller.sol

```diff
@@ -359,8 +363,6 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
      * @param _fullPauser new fullPauser address
      */
     function setFullPauser(address _fullPauser) external onlyOwner {
-        require(_fullPauser != address(0), "C10");
```

Comment:
can we keep this require

Reply:
I tried before committing - we can't have C9 + C10 + C11 back - for lack of space 
Perhaps 1 or 2 out of the 3